style.css

```diff
@@ -0,0 +1,331 @@
+/* the + sign is used to select the sibling that comes immediately next to the element you selected.
```

Comment:
In this file there a some empty rows and the indentions isn't the same everywhere. It would be great if the whole file would behave the same!